index.html

```diff
@@ -757,16 +756,11 @@ <h2>
           </li>
           <li>Set |subscription|'s {{PushSubscription/options}} attribute to |options|.
           </li>
-          <li>Generate a new P-256 <a>ECDH</a> key pair [[ANSI-X9-62]]. Store the private key in an
-          internal slot on |subscription|; this value MUST NOT be made available to applications.
-          The public key is also stored in an internal slot and can be retrieved by calling the
-          {{PushSubscription/getKey()}} method of the {{PushSubscription}} with an argument of
-          {{PushEncryptionKeyName/"p256dh"}}.
+          <li>Set |subscription|'s [=P-256 ECDH key pair=] to the result of generating a new P-256
+          [=ECDH=] key pair [[ANSI-X9-62]].
           </li>
-          <li>Generate a new authentication secret, which is a sequence of octets as defined in
-          [[RFC8291]]. Store the authentication secret in an internal slot on |subscription|. This
-          key can be retrieved by calling the {{PushSubscription/getKey()}} method of the
-          {{PushSubscription}} with an argument of {{PushEncryptionKeyName/"auth"}}.
+          <li>Set |subscription|'s [=authentication secret=] to the result of generating a new
+          authentication secret, which is a sequence of octets as defined in [[RFC8291]].
```

Comment:
Same with this one... maybe @martinthomson can quickly check this one too for clarity?